frontend/src/api.ts

```diff
@@ -259,7 +259,9 @@ export function useScratch(slugOrUrl: string): {
     const setScratch = useCallback((partial: Partial<Scratch>) => {
         const scratch = Object.assign({}, localScratch, partial)
 
-        if (!dequal(scratch, savedScratch)) {
+        if (dequal(scratch, savedScratch)) {
```

Comment:
The reason for this was to not force a rerender if the saved scratch and the serverside scratch are already equal. It's weird to me that this would be the cause of a bug, and this "fix" certainly seems logically wrong.

Reply:
Try testing the live viewing of someone else editing a scratch, this change probably breaks it